pkg/validate/validate.go

```diff
@@ -257,6 +261,11 @@ func validateRouterProfile(rp api.RouterProfile) (errs []error) {
 		errs = append(errs, fmt.Errorf("invalid properties.routerProfiles[%q].fqdn %q", rp.Name, rp.FQDN))
 	}
 
+	// for private preview
+	if rp.PublicSubdomain != "" && rp.PublicSubdomain != rp.FQDN {
```

Comment:
This logic is incorrect and should be removed.  For PP I think we're going to have to rely on the RP invalidating any user-provided rp.PublicSubdomain and populating the value here itself.  @sozercan is this realistic?